python/servo/platform/build_target.py

```diff
@@ -293,8 +293,8 @@ def configure_build_environment(self, env: Dict[str, str], config: Dict[str, Any
                 meta = json.load(meta_file)
             ohos_api_version = int(meta["apiVersion"])
             ohos_sdk_version = parse_version(meta["version"])
-            if ohos_sdk_version < parse_version("5.0") or ohos_api_version < 12:
-                raise RuntimeError("Building servo for OpenHarmony requires SDK version 5.0 (API-12) or newer.")
+            if ohos_sdk_version < parse_version("5.0") or ohos_api_version < 14:
+                raise RuntimeError("Building servo for OpenHarmony requires SDK version 5.0.2 (API-14) or newer.")
```

Comment:
Please also open a PR over at the servo book and document that we now need this version or newer.